docs/docs/angular.md

```diff
@@ -7,6 +7,11 @@ sidebar_label: Angular Integration
 
 [![npm version](https://badge.fury.io/js/mini-rx-store-ng.svg)](https://www.npmjs.com/package/mini-rx-store-ng)
 
+:::tip
+For modern Angular (with Signals and standalone) we recommend to use the [MiniRx Signal Store](https://github.com/spierala/mini-rx-store/blob/master/libs/signal-store/README.md).
```

Comment:
Perhaps add a link to https://angular.dev/guide/signals behind the first 'Signals'?

Reply:
goed plan!